chain/epoch_manager/README.md

```diff
@@ -0,0 +1,40 @@
+# near-epoch-manager crate
+
+Epoch manager crate is responsible for code related to epochs and epoch switching.
+An epoch is a unit of time when validators of the network remain constant.
+
+You can read more about the epoch here: https://docs.near.org/docs/concepts/epoch
+
+You can read more about Epoch finalization and Epoch changes here: https://github.com/near/NEPs/blob/master/specs/BlockchainLayer/EpochManager/EpochManager.md 
+
+## EpochManager
+Main class that has two main functions:
+* it creates new epochs (EpochIds)
+* allows accessing information about past and current epochs.
```

Comment:
I'd maybe start with more explicit list of what is the actual data? 

Ie, epoch is the set of validators, plus hashes of boundary blocks, etc.

Reply:
done. BTW - I don't think that epoch manager returns hashes of boundary blocks (at least I didn't see it)